playwright/components/progress-tracker/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Page } from "@playwright/test";
+import { PROGRESS_TRACKER, PROGRESS_TRACKER_LINE } from "./locators";
+
+// component preview locators
+export const progressTrackerComponent = (page: Page) =>
+  page.locator(PROGRESS_TRACKER);
+
+export const progressTrackerLine = (page: Page) =>
+  page.locator(PROGRESS_TRACKER_LINE);
+
+export const progressTrackerMinVal = (page: Page, index = 0) =>
+  progressTrackerComponent(page).locator("* > span").nth(index);
+
+export const progressTrackerMaxVal = (page: Page, index = 2) =>
+  progressTrackerComponent(page).locator("* > span").nth(index);
+
+export const progressTrackerCustomValuePreposition = (page: Page, index = 1) =>
+  progressTrackerComponent(page).locator("* > span").nth(index);
+
+export const progressTrackerDescription = (page: Page, index = 3) =>
+  progressTrackerComponent(page).locator("* > span").nth(index);
```

Comment:
**suggestion(non-blocking)**: similar to the data tag we've added for the inner bar, what do you think about us doing the same for the individual labels?

That way we wouldn't need to pass the `index` parameter to these label locators and cut down on the complexity.